packages/extension/src/newtab/MainFeedPage.tsx

```diff
@@ -8,6 +8,10 @@ import FeedLayout from '@dailydotdev/shared/src/components/FeedLayout';
 import dynamic from 'next/dynamic';
 import AuthContext from '@dailydotdev/shared/src/contexts/AuthContext';
 import AlertContext from '@dailydotdev/shared/src/contexts/AlertContext';
+import { useFeature } from '@dailydotdev/shared/src/components/GrowthBookProvider';
```

Comment:
Would this impact enrollment time for extension users? (When they first get enrolled)

Reply:
The current process is Boot -> MainLayout -> this file. So there should be no impact.